src/components/campaigns/grid/CampaignGrid.tsx

```diff
@@ -50,19 +50,21 @@ const DisplayExpandableDescription = (params: GridRenderCellParams<string>) => {
 }
 
 const DisplayReachedAmount = ({ params }: CampaignCellProps) => {
-  const summary = params.row.summary.find(() => true)
```

Comment:
The old implementation was correct, reached amount should be coming from the summary prop

Reply:
https://github.com/podkrepi-bg/api/blob/129ca8fbe42483a242f9463aff67c69b843682e4/apps/api/src/campaign/campaign.service.ts#L468-L505
if i use this function to pin on the summary prop. it will be not possible to display blocked amount, because returns empty vaults list.